jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -1424,6 +1424,33 @@
     # END KUBEMARK
 
     # START UPGRADE
+
+    ### Latest version upgrades #krousey
+    - kubernetes-e2e-gce-latest-upgrade-master: # upgrade master from previous version to latest
+        job-name: ci-kubernetes-e2e-gce-latest-upgrade-master
+        timeout: 70
```

Comment:
This timeout is the bootstrap wrapper timeout which needs to be longer than e2e.go timeout which is KUBEKINS_TIMEOUT(90m). Generally we give +20m here to give bootstrap some space to set up repo & upload logs.